fw/dilemma-features.md

```diff
@@ -0,0 +1,163 @@
+
+
+Custom defines (with default values):
+
+```
+#define Dilemma_MINIMUM_DEFAULT_DPI 400
+#define Dilemma_DEFAULT_DPI_CONFIG_STEP 200
+```
+
+Custom functions:
+
+```c
+dilemma_cycle_pointer_default_dpi(bool forward) // cycle forward or backward the possible values
+dilemma_cycle_pointer_default_dpi_noeeprom(bool forward) // cycle forward or backward the possible values without persisting the change to EEPROM
+dilemma_get_pointer_default_dpi() // returns the current DPI value
+```
+
+
+### Sniping
+
+**Sniping mode** slows down the pointer for more precise gestures. It is useful when combined with a higher default DPI. Like the default pointer's DPI, the sniper mode DPI can be changed at runtime
+
+Custom Keycodes:
+
+| Name   | Description                                                                  |
+| ------ | ---------------------------------------------------------------------------- |
+| `S_D_MOD` | increase the sensitivity of the pointer movement in sniping mode by one step |
+| `S_D_RMOD` | decrease the sensitivity of the pointer movement in sniping mode by one step |
+| `SNIPING`  | enable sniping mode as long as the key is pressed                            |
+| `SNP_TOG` | toggle sniping mode on and off                                               |
+
+
+Custom defines (with default values):
+
+```
+#define DILEMMA_MINIMUM_SNIPING_DPI 200
+#define DILEMMA_SNIPING_DPI_CONFIG_STEP 100
+```
+
+Custom functions:
+
+```c
+dilemma_set_pointer_sniping_enabled(bool enable) // enable/disable sniping mode
+dilemma_get_pointer_sniping_enabled() // returns whether sniping mode is currently enabled
+dilemma_cycle_pointer_sniping_dpi(bool forward) // cycle forward or backward the possible values
+dilemma_cycle_pointer_sniping_dpi_noeeprom(bool forward) // cycle forward or backward the possible values without persisting the change to EEPROM
+dilemma_get_pointer_sniping_dpi() // returns the current sniping mode DPI value
+```
+
+### Auto sniping on layer
+
+You can trigger sniping automatically when on a specific layer by adjusting the following in your keymap:
+
+```
+#define DILEMMA_AUTO_SNIPING_ON_LAYER LAYER_POINTER
+```
+
+### Auto pointer layer
+
+You can trigger the pointer layer automatically upon moving the trackpad by adjusting the following in your keymap:
+
+```
+#define DILEMMA_AUTO_POINTER_LAYER_TRIGGER_ENABLE
+#define DILEMMA_AUTO_POINTER_LAYER_TRIGGER_TIMEOUT_MS 1000
+```
+
+### Drag-scroll
+
+**Drag-scroll** enables scrolling with the trackpad. When drag-scroll is enabled, the trackpad's `x` and `y` movements are converted into `h` (horizontal) and `v` (vertical) movement, effectively sending scroll instructions to the host system.
+
+Custom keycodes:
+
+| Name   | Description                                           |
+| ------ | ----------------------------------------------------- |
+| `DRGSCRL`  | enable drag-scroll mode as long as the key is pressed |
```

Comment:
Could optionally mention the long name aliases

Reply:
iirc the long ones don't work in VIA, so let's keep it simple